tee-worker/omni-executor/rpc-server/src/methods/omni/README_HYPERLIQUID_RPC_EXAMPLES.md

```diff
@@ -0,0 +1,222 @@
+# Hyperliquid RPC Methods Usage Examples
+
+This document demonstrates how to use both the **separate** and **unified** approaches for Hyperliquid signature generation.
```

Comment:
Hmm why do we need both?

Can we choose one? The decision should mainly come from the client's side

Reply:
If we are sure they'll eventually be mapped into the same HL API call, then unified method should be preferred.

Maybe even ask the client to pass in the `Action` struct to be more generic